deployment/templates/services.yaml

```diff
@@ -9,4 +9,4 @@ spec:
   ports:
     - port: 8080
       targetPort: 8080
-      nodePort: 30008
+      nodePort: 30010
```

Comment:
If the nodePort has to be changed, please use a variable in Values.

Reply:
changed back to 30008